plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/preferences/OpenShiftPreferencePage.java

```diff
@@ -90,19 +90,38 @@ public void init(IWorkbench workbench) {
 	@Override
 	protected void performDefaults() {
 		String location = ocBinary.getLocation();
+		if(location == null) {
+			//We have to update default value in preferences even if it is empty.
+			location = "";
+		}
+		getPreferenceStore().setDefault(IOpenShiftCoreConstants.OPENSHIFT_CLI_LOC, location);
+
```

Comment:
Because otherwize OpenShiftCorePreferences.INSTANCE.getOCBinaryLocation() will return non-empty value and as soon as user changes $PATH, that value will not be valid.